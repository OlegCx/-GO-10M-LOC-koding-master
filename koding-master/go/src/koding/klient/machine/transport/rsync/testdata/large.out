receiving incremental file list
cd+++++++++ test/
>f+++++++++ test/file.1
              0   0%    0.00kB/s    0:00:00          934,234   7%  902.41kB/s    0:00:12        1,917,770  15%  876.38kB/s    0:00:11        3,032,362  24%  939.79kB/s    0:00:09        3,622,474  29%  844.89kB/s    0:00:10        4,507,632  36%  824.59kB/s    0:00:09        5,261,602  42%  793.74kB/s    0:00:08        6,015,612  48%  706.09kB/s    0:00:08        6,671,261  54%  720.03kB/s    0:00:07        7,458,073  60%  700.02kB/s    0:00:06        8,113,673  66%  673.74kB/s    0:00:06        8,802,124  71%  656.50kB/s    0:00:05        9,588,931  78%  684.10kB/s    0:00:03       10,342,922  84%  677.22kB/s    0:00:02       11,195,292  91%  727.26kB/s    0:00:01       12,080,423  98%  769.21kB/s    0:00:00       12,288,000 100%  759.54kB/s    0:00:15 (xfr#1, to-chk=3/5)
>f+++++++++ test/file.2
              0   0%    0.00kB/s    0:00:00          835,958  13%  809.08kB/s    0:00:06        2,180,069  35%    1.02MB/s    0:00:03        3,556,915  57%    1.12MB/s    0:00:02        5,196,041  84%    1.22MB/s    0:00:00        6,144,000 100%    1.27MB/s    0:00:04 (xfr#2, to-chk=2/5)
>f+++++++++ test/file.3
              0   0%    0.00kB/s    0:00:00          721,167   4%  696.60kB/s    0:00:22        2,393,077  14%    1.13MB/s    0:00:12        4,196,076  25%    1.33MB/s    0:00:08        5,769,662  35%    1.37MB/s    0:00:07        7,556,421  46%    1.62MB/s    0:00:05        9,130,030  55%    1.53MB/s    0:00:04       10,539,682  64%    1.44MB/s    0:00:03       12,047,731  73%    1.43MB/s    0:00:02       13,621,295  83%    1.38MB/s    0:00:01       15,194,806  92%    1.43MB/s    0:00:00       16,384,000 100%    1.43MB/s    0:00:10 (xfr#3, to-chk=1/5)
>f+++++++++ test/file.4
              0   0%    0.00kB/s    0:00:00          639,213   5%  610.20kB/s    0:00:19        2,442,341  19%    1.15MB/s    0:00:08        4,114,146  33%    1.29MB/s    0:00:06        5,720,633  46%    1.35MB/s    0:00:04        7,130,264  58%    1.53MB/s    0:00:03        8,834,889  71%    1.51MB/s    0:00:02       10,474,048  85%    1.49MB/s    0:00:01       11,949,229  97%    1.47MB/s    0:00:00       12,288,000 100%    1.41MB/s    0:00:08 (xfr#4, to-chk=0/5)

sent 104 bytes  received 47,131,119 bytes  1,163,733.90 bytes/sec
total size is 47,104,000  speedup is 1.00
