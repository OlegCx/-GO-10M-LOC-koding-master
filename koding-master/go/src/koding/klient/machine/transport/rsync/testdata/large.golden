Files: 2, Size 0
Files: 2, Size 934234
Files: 2, Size 1917770
Files: 2, Size 3032362
Files: 2, Size 3622474
Files: 2, Size 4507632
Files: 2, Size 5261602
Files: 2, Size 6015612
Files: 2, Size 6671261
Files: 2, Size 7458073
Files: 2, Size 8113673
Files: 2, Size 8802124
Files: 2, Size 9588931
Files: 2, Size 10342922
Files: 2, Size 11195292
Files: 2, Size 12080423
Files: 2, Size 12288000
Files: 3, Size 12288000
Files: 3, Size 13123958
Files: 3, Size 14468069
Files: 3, Size 15844915
Files: 3, Size 17484041
Files: 3, Size 18432000
Files: 4, Size 18432000
Files: 4, Size 19153167
Files: 4, Size 20825077
Files: 4, Size 22628076
Files: 4, Size 24201662
Files: 4, Size 25988421
Files: 4, Size 27562030
Files: 4, Size 28971682
Files: 4, Size 30479731
Files: 4, Size 32053295
Files: 4, Size 33626806
Files: 4, Size 34816000
Files: 5, Size 34816000
Files: 5, Size 35455213
Files: 5, Size 37258341
Files: 5, Size 38930146
Files: 5, Size 40536633
Files: 5, Size 41946264
Files: 5, Size 43650889
Files: 5, Size 45290048
Files: 5, Size 46765229
Files: 5, Size 47104000
